services/user/Tokens/ui/src/lib/createIdenticon.ts

```diff
@@ -0,0 +1,12 @@
+import { createAvatar } from "@dicebear/core";
+import { identicon } from "@dicebear/collection";
```

Comment:
I'd like to propose we use the dicebear (glass)[https://www.dicebear.com/playground/?style=glass] collection instead of identicons. No transparency to deal with, and it's a smoother feel.